OsuPlayer/Program.cs

```diff
@@ -96,6 +98,7 @@ private static void Register(IMutableDependencyResolver services, IReadonlyDepen
             statisticsProvider: resolver.GetRequiredService<IStatisticsProvider>(),
             sortProvider: resolver.GetRequiredService<ISortProvider>(),
             historyProvider: resolver.GetRequiredService<IHistoryProvider>(),
+            discordService: resolver.GetRequiredService<IDiscordService>(),
```

Comment:
Please only use GetService for the optional nullable services in the player. Also change this for the other ones that are the same here.